### ISSUE TITLE
Unable to export 

### ISSUE THREAD

**Issue by srileo** [Instrument/cyclops#3]
Hello all,
Am unable to Export the selected properties from AE. Inspite of selecting location, I do not see my exported file. I am running AE 2015 CC on a mac.

**Comment by ogmischka** [comment]
same problem here. 
does this plugin work with cc?

**Comment by srileo** [comment]
ogmischka, I checked in with the makers of the plugin. They closed shop and moved on. It doesn't work and is not supported. 
Let me know if you come up with a workaround. 
Also, do let me know if you know of a GUI tool that creates CSS animations.

**Comment by ghost** [comment]
Yes, exactly i need it too. If you learn please let me know too.
Also i called to company they said to me that we closed that project.

On Wed, May 11, 2016 at 10:57 AM, srileo notifications@github.com wrote:

> ogmischka, I checked in with the makers of the plugin. They closed shop
> and moved on. It doesn't work and is not supported.
> Let me know if you come up with a workaround.
> Also, do let me know if you know of a GUI tool that creates CSS
> animations.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/Instrument/cyclops/issues/3#issuecomment-218538839

## 

Güven AGAS
